cli/src/main/resources/hudson/cli/client/Messages.properties

```diff
@@ -3,7 +3,7 @@ CLI.Usage=Jenkins CLI\n\
   Options:\n\
    -s URL       : the server URL (defaults to the JENKINS_URL env var)\n\
    -i KEY       : SSH private key file used for authentication\n\
-   -p HOST:PORT : HTTP proxy host and port for HTTPS proxy tunneling. See http://jenkins-ci.org/https-proxy-tunnel\n\
```

Comment:
@kohsuke Apparently ([WEBSITE-319](https://issues.jenkins-ci.org/browse/WEBSITE-319)) this URL never existed. Any idea where this was supposed to go? Or did this documentation never exist?